miden-crypto/benches/README.md

```diff
@@ -25,6 +25,7 @@ We benchmark the above hash functions using two scenarios. The first is a 2-to-1
 | Amazon Graviton 3   | 108 ns |        |           |           | *5.3 µs  | *3.1 µs  |
 | Amazon Graviton 4   | 96 ns  |        |           |           | *5.1 µs  | *2.8 µs  |
 | AMD Ryzen 9 5950X   | 64 ns  | 273 ns |           |           | *5.5 µs  |          |
```

Comment:
Should we get rid of `AMD Ryzen 9 5950X` row? I think of `AMD Ryzen 9 9950X` as of basically replacing this benchmark.